pkg/catalog/retry_test.go

```diff
@@ -67,19 +74,19 @@ func TestGetRetryPolicy(t *testing.T) {
 						},
 						RetryPolicy: policyV1alpha1.RetryPolicySpec{
 							RetryOn:                  "5xx",
-							PerTryTimeout:            "2ns",
-							NumRetries:               3,
-							RetryBackoffBaseInterval: "4s",
+							PerTryTimeout:            &thresholdDuration,
+							NumRetries:               &thresholdUintVal,
+							RetryBackoffBaseInterval: &thresholdDuration,
```

Comment:
Having distinct values for `PerTryTimeout` and `RetryBackoffBaseInterval` would be an easy way to make sure these aren't getting transposed anywhere.